src/pkg/DistributionXml.cpp

```diff
@@ -128,9 +128,16 @@ bool DistributionXml::package(const std::string& id, DistributionXml::PkgRef& pk
 
 		if (pkg.path.empty() && node->children && node->children->type == XML_TEXT_NODE && !xmlIsBlankNode(node->children))
 			pkg.path = (char*) node->children->content;
+
+		std::string encodedSpaceTerm = "%20";
+		size_t spacePosition = pkg.path.find(encodedSpaceTerm);
+
+		while (spacePosition != std::string::npos) {
+			pkg.path.replace(spacePosition, 3, " ");
+			spacePosition = pkg.path.find(encodedSpaceTerm);
+		}
```

Comment:
It would be better if we rely on a method to handle decoding the URL instead. Otherwise, [we would have to manually deal with converting all of the `%` encodings](https://en.wikipedia.org/wiki/Percent-encoding#Reserved_characters)

One idea is to use `NSUrl` (we can convert the file into a objective-C++ file)